DepositeCalcTests/Pages/RegisterPage.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Threading;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class RegisterPage : BasePage
+    {
+        public RegisterPage(IWebDriver driver) : base(driver)
+        {
+        }
+
+        private IWebElement LoginFld => driver.FindElement(By.XPath("//input[@id='login']"));
+        private IWebElement EmailFld => driver.FindElement(By.XPath("//th[text()='Email:']/..//input"));
+        private IWebElement PassworldFld => driver.FindElement(By.XPath("//th[text()='Password:']/..//input"));
+        private IWebElement ConfirmPasswordFld => driver.FindElement(By.XPath("//th[text()='Confirm password:']/..//input"));
+        private IWebElement ReturnToLoginLnk => driver.FindElement(By.XPath("//div[@onclick='Logout()']"));
+        private IWebElement RegisterBtn => driver.FindElement(By.XPath("//button[@id='register']"));
+
+        public LoginPage ReturnToLogin()
+        {
+            ReturnToLoginLnk.Click();
+            return new LoginPage(driver);
+        }
+
+        public string ErrorMessage
+        {
+            get
+            {
+                var locator = By.Id("errorMessage");
+                IWebElement mainErrMessage = driver.FindElement(locator);
+                new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => mainErrMessage.Text.Length > 0);
+                return mainErrMessage.Text;
+            }
+        }
+
+        public bool IsOpened()
+        {
+            bool isConfirmPasswordFldPresent = driver.FindElements(By.XPath("//th[text()='Confirm password:']/..//input")).Count > 0;
+            return isConfirmPasswordFldPresent;
```

Comment:
You can simplify it a little. 'Count > 0' can be replaced with 'Any()', but maybe you should add 'using System.Linq;' 
```suggestion
            return driver.FindElements(By.XPath("//th[text()='Confirm password:']")).Any();
```